Patch to add Solaris specific elements to the main Apache
configuration file.  This patch has not yet been submitted upstream.

--- puppetlabs-apache-1.8.1/manifests/init.pp.orig	2016-04-27 07:52:17.681437269 -0700
+++ puppetlabs-apache-1.8.1/manifests/init.pp	2016-04-27 07:52:49.540772562 -0700
@@ -1,3 +1,7 @@
+#######################################################################
+# Oracle has modified the originally distributed contents of this file.
+#######################################################################
+
 # Class: apache
 #
 # This class installs Apache
@@ -305,6 +309,11 @@
         $scriptalias          = '/usr/lib/cgi-bin'
         $access_log_file      = 'access.log'
       }
+      'solaris': {
+        $error_log            = 'error_log'
+        $scriptalias          = '/var/apache2/2.4/cgi-bin/'
+        $access_log_file      = 'access_log'
+      }
       default: {
         fail("Unsupported osfamily ${::osfamily}")
       }
